environment.json

```diff
@@ -1,13 +1,11 @@
 {
   "protoc": {
-    "PATH": "../bin/protoc"
+    "PATH": "/usr/local/bin"
   },
   "verilator": {
-    "PATH": "/usr/local/bin",
-    "VERILATOR_ROOT": "../verilator/share/verilator"
+    "PATH": "/usr/local/bin"
   },
   "riscv-toolchain": {
-    "PATH": "../opt/riscv64-unknown-elf-gcc-8.2.0-2019.05.3-x86_64-linux-ubuntu14/bin",
-    "RISCV": "../opt/riscv64-unknown-elf-gcc-8.2.0-2019.05.3-x86_64-linux-ubuntu14/bin"
```

Comment:
why arent' you setting `RISCV`?

Reply:
It didn't seem to be required for getting the PIO block tests compiled and running. It also looked a little bit wrong in the old version of this JSON file because I thought the `RISCV` environment variable used in other places like rocket-chip was supposed to point to the directory one level higher. For example: https://github.com/chipsalliance/rocket-chip/blob/3e2c304a782ef9a0d06b8cfab4c78f49f9fc1674/scripts/debug_rom/Makefile#L5

I can put it back in, but do you know if it should point to the `bin/` directory or the one above it?